### PR TITLE
DOI Documentation

### PR DESCRIPTION
**Pull request by Kakulukian**
I began to write the documentation for DOI generation.

Mainly based on the blogpost

### CODE REVIEW

**Review by julien-c — COMMENTED** [review]
looks good to me!

Will need to update the "DOI" section in repo settings to point to this doc, once it's up
Will need to update the blogpost to also point to this doc

Will need to link to the blogpost announcement from this doc

**Comment by Kakulukian** [comment]
cc @sashavor @cakiki

**Comment by Kakulukian** [comment]
Isn't it more like when the cookie expires? It seems to be done at logout

File: docs/hub/doi.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+After you agree to those terms, your model/dataset will get a DOI assigned, and a new tag should appear in your model/dataset header allowing you to cite it.
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/doi-header-with-doi.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/doi-header-with-doi-dark.png"/>
+</div>
+
+
+## Can I regenerate a new DOI if my model/dataset changes?
```
Comment (mishig25) [inline docs/hub/doi.md:28]: could you try replacing `model/dataset` with `model or dataset` here ?

**Review by osanseviero — APPROVED** [review]
The content looks good!

File: docs/hub/doi.md [hunk 2]
```diff
@@ -36,9 +36,10 @@ If ever there’s a new version of a model or dataset, a new DOI can easily be a
 
 You just need to click on "Generate new DOI" and tadaam!🎉 a new DOI is assigned for the current revision of your model or dataset.
 
-## Why there is `locked by DOI` message on delete, rename and change visibility action on my model or dataset?
+## Why there is 'locked by DOI' message on delete, rename and change visibility action on my model or dataset?
```
Comment (cakiki) [inline docs/hub/doi.md:39]: Small nit: should be "Why is there"

**Review by cakiki — CHANGES_REQUESTED** [review]

File: docs/hub/doi.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Digital Object Identifier (DOI)
+
+The Hugging Face Hub offers the possibility to generate DOI for your models or datasets. DOIs (Digital Object Identifiers) are strings uniquely identifying a digital object, anything from articles to figures, including datasets and models. DOIs are tied to object metadata, including the object's URL, version, creation date, description, etc. They are a commonly accepted reference to digital resources across research and academic communities; they are analogous to a book's ISBN.
+
+## How to generate a DOI?
+
+To do this, you must go to the settings of your model or dataset. Then you have to go to the DOI section, a button called "Generate DOI" should appear:
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/doi-generation.png"/>
```
Comment (cakiki) [inline docs/hub/doi.md:10]: Why is the "to" highlighted?

**Comment by Kakulukian** [comment]
Thanks @cakiki 👍 
I merge it!

**Pull request merged**